doc/source/reference/commands/status.rst

```diff
@@ -147,3 +149,39 @@ values:
    * - ``version``
      - The version of the Groonga process.
      - ``5.0.7``
+
+   * - ``features``
+     - .. versionadded:: 10.0.1
+     
+       The list of Groonga's features and status(enabled or disabled).
```

Comment:
@yoshimotoyuk Should we put a space before `(`?

```suggestion
       The list of Groonga's features and status (enabled or disabled).
```

Reply:
yes. 
Sorry about it .